lib/ansible/config/module_defaults.yml

```diff
@@ -522,15 +522,15 @@ groupings:
   - azure
   azure_rm_aks:
   - azure
-  azure_rm_aks_facts:
+  azure_rm_aks_info:
```

Comment:
Please also retain the previous name since module_defaults doesn't set aliases.

Reply:
ok, thanks.

Reply:
I'd still add the _info modules to the list. Just keep the _facts name alternatives there too.

Reply:
The `_info` modules are now missing. Will they be added to `module_defaults.yml` in another PR?